sdk/avalon_sdk/worker/worker_request_validator.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from utility.hex_utils import is_valid_hex_str
+import base64
+import binascii
+
+
+class WorkerRequestValidator():
+    """
+    WorkerRequestValidator validates worker related requests
+    for proper parameter fields and valid data formats.
+    """
+    def __init__(self):
+        """
+        Initialize __param_key_map and __value_key_map,
+        which are key-value pairs.
+        The key is the field name and value is a boolean that
+        indicates whether a field is mandatory (True) or optional (False).
+        """
```

Comment:
checkout jsonschema validation -> https://python-jsonschema.readthedocs.io/en/latest/

Reply:
done

Reply:
reverted the usage of jsonschema

Reply:
Are you planning to raise a separate PR for jsonschema ?

Reply:
yes it involves too 2 library installations and document docker changes Will do it in seperate PR for all worker, workorder related files.